src/Core/Defines.h

```diff
@@ -69,10 +69,10 @@
 
 /// Mininum revision supporting interserver secret.
 /// TODO: sync with cmake/autogenerated_versions.txt before merging
-#define DBMS_MIN_REVISION_WITH_INTERSERVER_SECRET 54441
+#define DBMS_MIN_REVISION_WITH_INTERSERVER_SECRET 54440
 
 /// Version of ClickHouse TCP protocol. Set to git tag with latest protocol change.
-#define DBMS_TCP_PROTOCOL_VERSION 54441
+#define DBMS_TCP_PROTOCOL_VERSION 54226
```

Comment:
That should be assigned to the version with latest protocol change, i.e. 54440.
I know that this value unlikely affects anything, but it's better to change it anyway.